tests/unit/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "unit",
+  "version": "1.0.0",
+  "dependencies": {
+  }
+}
```

Comment:
What is this? Should we have it in the other projects?

Reply:
No, good call, it's not necessary.  I was trying something new, but this file shouldn't have been committed.